### PR TITLE
New package: python3-mpdnotify-1.0

### PR DESCRIPTION
**Pull request by anjandev**

### CODE REVIEW

File: srcpkgs/python3-mpdnotify/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'python3-mpdnotify'
+pkgname=python3-mpdnotify
+version=1.0
+revision=1
+wrksrc=mpdnotify-${version}
+build_style=python3-module
+hostmakedepends="python3-setuptools python3-Pillow python3-mpd2"
+depends="libnotify python3-mpd2 python3-Pillow"
```
Comment (Chocimier) [inline srcpkgs/python3-mpdnotify/template]: also python3-setuptools

File: srcpkgs/python3-mpdnotify/template [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+short_desc="Notifies when mpd's song changes"
+maintainer="Anjandev Momi <anjan@momi.ca>"
+license="GPL-3.0-or-later"
```
Comment (Chocimier) [inline srcpkgs/python3-mpdnotify/template]: -only

File: srcpkgs/python3-mpdnotify/template (hunk 1 above)
Comment (anjandev) [reply srcpkgs/python3-mpdnotify/template]: Why do I need setuptools as a runtime dependency?

Comment (Chocimier) [reply srcpkgs/python3-mpdnotify/template]: Well, it won't run without, you can check within xbps-src chroot.

I am not sure why setuptools started to inject inself into all it builds, maybe this should be prevented.

**Pull request merged**